hub-saml/src/test/java/uk/gov/ida/saml/core/validators/assertion/DuplicateAssertionValidatorTest.java

```diff
@@ -38,7 +40,8 @@ public void setUp() {
         duplicateIds.put("duplicate", DateTime.now().plusMinutes(5));
         duplicateIds.put("expired-duplicate", DateTime.now().minusMinutes(2));
 
-        duplicateAssertionValidator = new DuplicateAssertionValidatorImpl(duplicateIds);
+        IdExpirationCache idExpirationCache = new ConcurrentMapIdExpirationCache<>(duplicateIds);
```

Comment:
Is `new ConcurrentMapIdExpirationCache<>(duplicateIds);` same as  `IdExpirationCache<String>`? 🤔